server/routes/documents/pdf/letter-generator.js

```diff
@@ -199,7 +199,7 @@
           ],
           // these are important for the footer notes / text
           footerText: officer,
-          signature: data.signature,
+          signature: `${data.firstName} ${data.lastName}`,
```

Comment:
the signature is a string that we should get from the api that reads as `The jury officer` or `The court manager`... the `footerText` property is what sets the name of the officer printing the letter, which should also be driven by the api response....

Reply:
so here:

![image](https://github.com/hmcts/juror-bureau/assets/140706972/22c87694-e0e5-42e5-b956-23f8f6b0ee31)

`Court Manager` would be the name of the officer, that is set in the `footerText: ` property and `Jury Manager` would be the actual signature that we get from the api.